package-lock.json

```diff
@@ -10108,9 +10108,9 @@
       "integrity": "sha512-dRLjCWHYg4oaA77cxO64oO+7JwCwnIzkZPdrrC71jQmQtlhM556pwKo5bUzqvZndkVbeFLIIi+9TC40JNF5hNQ=="
     },
     "node_modules/@types/jsonwebtoken": {
-      "version": "8.5.9",
-      "resolved": "https://registry.npmjs.org/@types/jsonwebtoken/-/jsonwebtoken-8.5.9.tgz",
-      "integrity": "sha512-272FMnFGzAVMGtu9tkr29hRL6bZj4Zs1KZNeHLnKqAvp06tAIcarTMwOh8/8bz4FmKRcMxZhZNeUAQsNLoiPhg==",
+      "version": "9.0.1",
```

Comment:
this is a major version @alaister - are we sure it's OK?

Reply:
As said in the template it's a breaking change but reading the CHANGELOG (link in the PR template) seems a safe upgrade.

Reply:
@kiwicopple From what I can tell, we're not actually using the `jsonwebtoken` library anywhere, so maybe better to remove than upgrade?

Reply:
@alaister  `jsonwebtoken` can't be removed because it's a transitive dependency AFAIK.
```sh
$ npm why jsonwebtoken
jsonwebtoken@9.0.0
node_modules/jsonwebtoken
  jsonwebtoken@"^9.0.0" from universal-github-app-jwt@1.1.1
  node_modules/universal-github-app-jwt
    universal-github-app-jwt@"^1.0.1" from @octokit/auth-app@4.0.7
    node_modules/@octokit/auth-app
      @octokit/auth-app@"^4.0.0" from @octokit/app@13.0.11
      node_modules/@octokit/app
        @octokit/app@"^13.0.5" from octokit@2.0.10
        node_modules/octokit
          octokit@"^2.0.10" from www@0.0.3
          apps/www
            www@0.0.3
            node_modules/www
              workspace apps\www from the root project
```

Reply:
I'm updating our direct dependency on octokit instead. It is a patch release bump from 2.0.10 to 2.0.14 so there shouldn't be any breaking changes.